packages/api/src/utils/corsConfig.ts

```diff
@@ -9,5 +12,6 @@ export const corsConfig = {
     'https://web-demo.omnivore.app',
     'http://localhost:3000',
     'lsp://logseq.io',
+    env.dev.isLocal && 'https://studio.apollographql.com',
```

Comment:
@jacksonh I updated the code and now it will only allow Apollo sandbox access in local env

Reply:
I didn't realize that syntax works, cool